README.md

```diff
@@ -239,6 +226,9 @@ See `apps/test-logger` for an example of using the SWO-supported logger utility.
 3. Set the system clock rate to 180000000Hz and stimulus port to 0
 4. Click "Start". The SWO print statements should appear in the Serial Wire Viewer console.
 
+## Using the Debugger (SWO) (Rev 2 +)
+Detailed steps for this have not been created yet. Refer to this (link)[https://github.com/stlink-org/stlink/blob/develop/doc/tutorial.md] for guidance
```

Comment:
using gdb with stlink seems well documented. this will be  task for someone later on